minitest.el

```diff
@@ -64,14 +64,19 @@
   :type 'list
   :group 'minitest)
 
+(defcustom minitest-spring-command '("spring" "rake" "test")
+  "Spring command for minitest"
+  :type 'list
+  :group 'minitest)
+
 (defun minitest-buffer-name (file-or-dir)
   (concat "*Minitest " file-or-dir "*"))
 
 (defun minitest-test-command ()
-  (let ((command (cond (minitest-use-spring '("spring" "rake" "test"))
-                       ((minitest-zeus-p) '("zeus" "test"))
-                       (minitest-use-rails '("bin/rails" "test"))
-                       (t minitest-default-command))))
+  (let ((command (cond (minitest-use-spring minitest-spring-command)
+        ((minitest-zeus-p) '("zeus" "test"))
+        (minitest-use-rails '("bin/rails" "test"))
+        (t minitest-default-command))))
```

Comment:
Sorry to be nitpicky, I could just merge this in and fix it myself, but may as well pass on what little Elisp knowledge I have.  Conventional Lisp indentation would have these additional `cond` clauses lined up with the first one... underneath `(minitest-use-spring ...`  so basically this part of the diff should only show the change to the `minitest-use-spring` clause.  My Emacs automatically follows those indentation rules, but I'm not sure if that is standard or customized by me at this point ;)

Reply:
No problem, fixed now. My emacs copy indented it correctly as well. 😁